py/client/pydeephaven/_table_ops.py

```diff
@@ -610,30 +636,30 @@ def __init__(self, filter_table: Any, cols: List[str], inverted: bool):
         self.inverted = inverted
 
     @classmethod
-    def get_stub_func(cls, table_service_stub: table_pb2_grpc.TableServiceStub):
+    def get_stub_func(cls, table_service_stub: table_pb2_grpc.TableServiceStub) -> Any:
         return table_service_stub.WhereIn
 
-    def make_grpc_request(self, result_id, source_id=None):
+    def make_grpc_request(self, result_id, source_id) -> Any:
         right_id = table_pb2.TableReference(ticket=self.filter_table.ticket)
         return table_pb2.WhereInRequest(result_id=result_id,
                                         left_id=source_id,
                                         right_id=right_id,
                                         inverted=self.inverted,
                                         columns_to_match=self.cols)
 
-    def make_grpc_request_for_batch(self, result_id, source_id):
+    def make_grpc_request_for_batch(self, result_id, source_id) -> Any:
         return table_pb2.BatchTableRequest.Operation(
             where_in=self.make_grpc_request(result_id=result_id, source_id=source_id))
 
 
 class MetaTableOp(TableOp):
     @classmethod
-    def get_stub_func(cls, table_service_stub: table_pb2_grpc.TableServiceStub):
+    def get_stub_func(cls, table_service_stub: table_pb2_grpc.TableServiceStub) -> Any:
         return table_service_stub.MetaTable
 
-    def make_grpc_request(self, result_id, source_id):
+    def make_grpc_request(self, result_id, source_id) -> Any:
         return table_pb2.MetaTableRequest(result_id=result_id, source_id=source_id)
 
-    def make_grpc_request_for_batch(self, result_id, source_id):
+    def make_grpc_request_for_batch(self, result_id, source_id) -> Any:
```

Comment:
For this whole file, is `Any` really the best we can do?

Reply:
unfortunately yes.